pysb/importers/bngl.py

```diff
@@ -358,24 +378,34 @@ def _parse_rules(self):
     def _parse_expressions(self):
         expr_namespace = dict(
             self.model.parameters | self.model.expressions_constant()
-            | self.model.observables
```

Comment:
What happens with observables now?

Reply:
This removal was probably a mistake. Reverted.